api/filtering/db_custom_filters.py

```diff
@@ -396,3 +390,17 @@ def get_major_minor_from_version(version_split: list[str]):
     minor = version_split[0] if version_split else None
 
     return major, minor
+
+
+def create_os_filter(os_name, version_node, os_filter_list):
```

Comment:
`os_filter_list` doesn't need to be passed in; the output of this function can just be appended to the existing list
```suggestion
def create_os_filter(os_name, version_node):
    os_filter_list: list[OsFilter] = []
```